test/position-managers/PositionManager.t.sol

```diff
@@ -54,6 +56,23 @@ contract PositionManagerTest is Test, PosmTestSetup, LiquidityFuzzers {
         lpm.modifyLiquidities(abi.encode(planner.actions, badParams), block.timestamp + 1);
     }
 
+    function test_modifyLiquidities_reverts_reentrancy() public {
+        // Create a reentrant token and initialize the pool
+        ReentrantToken reentrantToken = new ReentrantToken(lpm);
+        (currency0, currency1) = (address(reentrantToken) < Currency.unwrap(currency1))
```

Comment:
could we underscore these vars.. just cause we also have them defined at the top level contract & same with `key`

Reply:
oh i was overriding them on purpose ... they reset their value at the beginning of every test 
but happy to bring in new vars if you want

Reply:
yeah its just more readable in the sense that we know they are different than the global params? but honestly its fine as is, its not a harmful test